QUARKUS-6263.md

```diff
@@ -0,0 +1,61 @@
+# QUARKUS-6263 - Add OIDC Health Check
+
+JIRA: https://issues.redhat.com/browse/QUARKUS-6263
+
+Upstream issues: https://github.com/quarkusio/quarkus/issues/47861
+
+Upstream PR: https://github.com/quarkusio/quarkus/pull/47830
+
+## Scope of the testing
+
+- OIDC Health Check is correctly registered when `quarkus.oidc.health.enabled=true` and `quarkus-smallrye-health` dependency is present
+- OIDC Health Check is not registered if `quarkus.health.extensions.enabled=false`
+- Individual tenant status testing:
+    - **OK**: When OIDC provider is available 
+    - **Disabled**: When `tenant-enabled=false`
```

Comment:
Please put there entire property name. Just `tenant-enabled=false` is confusing.